src/main/java/core/basesyntax/models/FruitTransaction.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.models;
+
+public class FruitTransaction {
+    public enum Operation {
+        SUPPLY("s"), PURCHASE("p"), RETURN("r");
```

Comment:
Why is FruitTransaction empty? Add fields, constructor
Add method to get Operation from code to enum